proposals/gc/Overview.md

```diff
@@ -574,7 +579,7 @@ Imported types are essentially parameters to the module.
 As such, they are entirely abstract, as far as compile-time validation is concerned.
 The only operations possible with them are those that do not require knowledge of their actual definition or size: primarily, passing and storing references to such types.
 
-TODO: The ability to import types makes the type and import sections interdependent.
+TODO: The ability to import types makes the type and import sections interdependent. (we will probably need to add placeholder entries in the types section for imported types, and possibly may need to add assumptions about their representation, even to compile code that uses them as locals, unless we want to restrict imported types to be only reference types).
```

Comment:
The assumptions could in fact be expressed as ordinary supertype bounds.

Not sure I follow the placeholder idea.